lib/nano.d.ts

```diff
@@ -248,7 +248,7 @@ declare namespace nano {
     /** whether to get a faster changes feed by supplying 'seq_interval'   */
     fastChanges?: boolean;
     /** where to begin the changes feed: 0, now or a sequence token */
-    since?: string;
+    since?: string | number;
```

Comment:
`0` is accepted so added `| number`.

Reply:
Yeah "since" is a bit of a nightmare isn't it? :)